fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -2822,23 +2822,23 @@ Table 45 contains a description of each different version of the **/services** r
 
 #### 6.11.2 Service types
 
-The following types of service are supported by the Mojaloop API
+The service types supported by the Mojaloop API are listed in the following sub-sections.
 
 ##### 6.11.2.1 Currency Conversion Services 
 
-A request for participants in a Mojaloop system who support currency conversion services (Foreign Exchange Providers or FXPs) is made using the syntax **services/FXP**
+A request for FSPs who support currency conversion services (that is, FSPs who have the rolw of Foreign Exchange Providers or FXPs) is made using the syntax **services/FXP**
```

Comment:
".. (that is, FSPs who have the rolw of Foreign Exchange Providers ..." rolw -> role